src/lib/mongoBackend/mongoSubCache.cpp

```diff
@@ -0,0 +1,1204 @@
+/*
```

Comment:
Appart from issue (which is ok), what do you think about adding *in this PR* hints about which functions to migrate? Have you a clear list of which are the candidate ones?